cmd/kraft/build/build.go

```diff
@@ -68,6 +68,18 @@ func New() *cobra.Command {
 	})
 }
 
+func (opts *Build) Pre(cmd *cobra.Command, args []string) error {
```

Comment:
_(putting my linter hat on)_ `opts` is unused. It's not a major issue, but if we don't remove it now, the `unused` linter (which I think we should enable) will eventually ask us to do so.

Reply:
Nice hat! 🤠 Are you saying this is possible?

```go
func (_ *Build) Pre(_ *cobra.Command, _ []string) error {
```

? I've never seen `func (_ *XXX)` before, but if this is possible, sure it make sense

Reply:
Just rename `opts` entirely.